.pre-commit-config.yaml

```
@@ -94,13 +94,8 @@ repos:
       - id: mypy
         additional_dependencies: [types-PyYAML, types-attrs, attrs]
         pass_filenames: false
-        files: "^python-sdk/"
+        files: "^python-sdk/|sql-cli/$"
```

```suggestion
        files: "^python-sdk/|^sql-cli/"
```
Why did you change this? Woudn't `sql-cli/$` only find the folder `sql-cli/` - but mypy has to run on files or?